mods/pvp/grenades/init.lua

```diff
@@ -2,15 +2,15 @@ grenades = {
 	grenade_deaccel = 9
 }
 
-local function throw_grenade(name, player)
+function grenades.throw_grenade(name, startvel, player)
 	local dir = player:get_look_dir()
 	local pos = player:get_pos()
 	local obj = minetest.add_entity({x = pos.x + dir.x, y = pos.y + 1.5 + dir.y, z = pos.z + dir.z}, name)
-
-	local m = 20
-	obj:set_velocity({x = dir.x * m, y = dir.y * m, z = dir.z * m})
+	obj:set_velocity({x = dir.x * startvel, y = dir.y * startvel, z = dir.z * startvel})
 	obj:set_acceleration({x = 0, y = -9.8, z = 0})
 
+	obj:get_luaentity().thrower_name = player:get_player_name()
+
 	return(obj:get_luaentity())
```

Comment:
Superfluous parenthesis - we don't use the `return()` syntax.

```suggestion
	return obj:get_luaentity()
```